src/screens/HospitalInformation.tsx

```diff
@@ -93,14 +97,44 @@ const HospitalInformation: React.FC<ScreenProps> = ({ navigation }) => {
           <View style={styles.variableButton}>
             <ButtonSecundary
               style={styles.variableButton}
-              title={
-                permission === "time-tracking"
-                  ? "Consultar Relatórios"
-                  : "Exportar Relatório"
-              }
+              title="Consultar Relatórios"
+              onPress={() => "nothingyet"}
```

Comment:
Essa parte do código deve ser mantida, é pras telas de gestão do app
```suggestion
             title={
                permission === "time-tracking"
                  ? "Consultar Relatórios"
                  : "Exportar Relatório"
              }
              onPress={() => "nothingyet"}
```